### PR TITLE
Bellis core 320

### PR DESCRIPTION
**Pull request by ellisbenjamin**
As a Namely Connect Admin
I want to see information about a company
So that I can make informed decisions when supporting them

--

Given I am on the installations list page
When I click a installations name
Then I see a page with the installation's enabled connections

https://namely.atlassian.net/browse/CORE-320

all cred to nick.

### CODE REVIEW

File: app/controllers/admin/installations_controller.rb [hunk 1]
```diff
@@ -20,7 +20,24 @@ def show
...
+      @installation = Installation.find(params[:id])
+      @connections = get_installation_connections(params[:id])
+    end
+  end
+
+  private
+
+  def get_installation_connections(i_id)
+    connections = []
+    INTEGRATIONS.each do |k,v|
```
Comment (johnnymo87) [inline app/controllers/admin/installations_controller.rb]: Would `map` work here?

File: app/controllers/admin/installations_controller.rb [hunk 2]
```diff
@@ -20,7 +20,24 @@ def show
           @installations << installation
         end
       end
+      render :index
+
+    else
+      @installation = Installation.find(params[:id])
```
Comment (cored) [inline app/controllers/admin/installations_controller.rb]: You could remove a little bit of duplication by having a method that do this lookup. 
```
def installation
@installation ||= Installation.find(params[:id])
end
```

File: config/routes.rb [hunk 3]
```diff
@@ -39,7 +39,6 @@
   )
 
   namespace :admin do
-    resources :installations
+    resources :installations, only: [:index, :show]
```
Comment (cored) [inline config/routes.rb]: Good call

**Comment by cored** [comment]
:+1:

File: app/controllers/admin/installations_controller.rb [hunk 4]
```diff
@@ -20,7 +20,24 @@ def show
...
+
+  private
+
+  def get_installation_connections(i_id)
+    connections = []
+    INTEGRATIONS.each do |k,v|
+      connection = Installation.find(i_id).connection_to(k)
+      if connection.connected?
+        connections << connection
+      end
```
Comment (bobbytables) [inline app/controllers/admin/installations_controller.rb]: We can do `each_with_object`, like so:

```ruby
INTEGRATIONS.each_with_object([]) do |(k, v), connections|
  connection = Installation.find(i_id).connection_to(k)
  if connection.connected?
    connections << connection
  end
end
```

**Comment by nnarh** [comment]
added @bobbytables and @cored suggestions with @ellisbenjamin

**Comment by bobbytables** [comment]
👍

**Pull request merged**